Octokit.Tests.Integration/Reactive/ObservableGistClientTests.cs

```diff
@@ -0,0 +1,720 @@
+﻿using System;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Octokit.Reactive;
+using Xunit;
+
+namespace Octokit.Tests.Integration.Reactive
```

Comment:
As mentioned below, can we drop this namespace from our new test class and re-indent this file?